src/Service/Factory/JwtServiceFactory.php

```diff
@@ -42,15 +43,22 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
             throw new RuntimeException('A verify key was not provided');
         }
 
-        $configuration = Configuration::forAsymmetricSigner($signer, InMemory::plainText($config['signKey']), InMemory::plainText($config['verifyKey']));
+        $signKey = $config['signKey'];
+        if (!is_object($signKey) || !$signKey instanceof Key) {
+            $signKey = InMemory::plainText($signKey);
+        }
+
+        $verifyKey = $config['verifyKey'];
```

Comment:
Do we need to check both or could we have one if for is_object then another if inside for instanceof?

Reply:
Normally we check both to keep phpstan happy, but since we don't use phpstan in this repo I think we can get away with just the `!$signKey instanceof Key` check. I'm not a fan of nested ifs though, if we can do without them

Reply:
Cool thats fine just thinking if it would made it slightly fast but of course np

Reply:
Nesting the ifs won't be any faster, because PHP doesn't evaluate the condition after `||` if the first condition equates to `true`